javascripts/discourse/components/dc-show-more.js.es6

```diff
@@ -0,0 +1,33 @@
+import Component from "@ember/component";
+import { action } from "@ember/object";
+
+export default Component.extend({
+  tagName: "section",
+  classNames: ["show-more-section"],
+  classNameBindings: ["isExpanded:expanded:collapsed"],
+  isExpanded: false,
+  displayText: "",
+
+  init() {
+    this._super(...arguments);
+    this._updateDisplayText();
+  },
+
+  willUpdate() {
+    this._updateDisplayText();
+  },
+
+  @action
+  toggleExpanded() {
+    this.set("isExpanded", !this.isExpanded);
+  },
+
+  _updateDisplayText() {
```

Comment:
@duranmla How this will work when we have multiple topics that are fetched as we scroll to the bottom (infinite scroll). In the design, I remove this button entirely after pressing it.

Thoughts on this?

Reply:
I think you are right, I should remove the button once the user expand so the built-in behaviour doesn't get affected _(infinite scroll)_ I didn't though on that when I implement this. _I will update._

Reply:
Addressed at b011891